lib/pbench/test/unit/server/test_requests.py

```diff
@@ -312,6 +399,82 @@ def test_upload(
         assert dataset.state == States.UPLOADED
         assert Metadata.getvalue(dataset, "dashboard") is None
         assert Metadata.getvalue(dataset, Metadata.DELETION) == "1972-01-01"
+        assert self.filetree_created
+        assert dataset.name in self.filetree_created
```

Comment:
If the assertion at line 402 would fail, then the assertion at line 403 would also fail, so I think you can delete line 402.

Reply:
That's a bit pessimistic (😉), but I support that.  😄